modules/logging-forward-splunk.adoc

```diff
@@ -31,34 +30,38 @@ $ oc -n openshift-logging create secret generic vector-splunk-secret --from-lite
 +
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: openshift-logging
 spec:
-  serviceAccountName: <service_account_name> <3>
+  managementState: Managed
+  serviceAccountName: <service_account_name> #<1>
   outputs:
-    - name: splunk-receiver <4>
-      secret:
-        name: vector-splunk-secret <5>
-      type: splunk <6>
-      url: <http://your.splunk.hec.url:8088> <7>
-  pipelines: <8>
-    - inputRefs:
+    - name: splunk-receiver #<2>
+      type: splunk #<3>
+      splunk:
+        authentication:
+          token:
+            key: hecToken # <4>
+            secretName: splunk-secret
+        url: '<http://your.splunk.hec.url:8088>' #<5>
+        index: '{.log_type||"undefined"}' #<6>
+  pipelines:
+    - name: my-logs
+      inputRefs: #<7>
         - application
         - infrastructure
-      name: <9>
       outputRefs:
-        - splunk-receiver <10>
+        - splunk-receiver #<8>
+
 ----
-<1> In legacy implementations, the CR name must be `instance`. In multi log forwarder implementations, you can use any name.
-<2> In legacy implementations, the CR namespace must be `openshift-logging`. In multi log forwarder implementations, you can use any namespace.
-<3> The name of your service account. The service account is only required in multi log forwarder implementations if the log forwarder is not deployed in the `openshift-logging` namespace.
-<4> Specify a name for the output.
-<5> Specify the name of the secret that contains your HEC token.
-<6> Specify the output type as `splunk`.
-<7> Specify the URL (including port) of your Splunk HEC.
-<8> Specify which log types to forward by using the pipeline: `application`, `infrastructure`, or `audit`.
-<9> Optional: Specify a name for the pipeline.
-<10> Specify the name of the output to use when forwarding logs with this pipeline.
+<1> The name of your service account.
+<2> Specify a name for the output.
+<3> Specify the output type as `splunk`.
+<4> Specify the name of the secret that contains your HEC token.
+<5> Specify the URL (including port) of your Splunk HEC.
+<6> Use the default index of the splunk server configuration is used when 'index' is not defined.
+<7> Specify which log types to forward by using the pipeline: `application`, `infrastructure`, or `audit`.
```

Comment:
same comment about `inputRefs` again, its not designed for log type, but rather the reference to the input type